### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by paulmuia**
added home page , ballot containers and assets

### CODE REVIEW

File: lib/pages/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+class HomePage extends StatelessWidget {
+  
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold (
+      appBar: AppBar(
+        backgroundColor: const Color.fromARGB(255, 2, 35, 63),
+        title: const Row(
```
Comment (Piexie3) [inline lib/pages/home_page.dart]:
```suggestion
        title: Text("Your vote your choice"),
        actions: [
        // add icons incide here
        Icon(Icons.verified_user_outlined)
        ]
```

**Review by Piexie3 — COMMENTED** [review]
Make sure you formart your code before you push 
In linux = ctrl+shft+i
In windows = shift+alt+f

**Comment by paulmuia** [comment]
formatted code

**Comment by paulmuia** [comment]
Edited appBar in home Page screen

**Review by Piexie3 — APPROVED** [review]
Also make sure to attach screenshots next time after creating a pull request

**Pull request merged**